app/services/fastboot.js

```diff
@@ -40,6 +40,8 @@ export default Ember.Service.extend({
     return this._fastbootInfo.request.host();
   }),
 
+  response: alias('_fastbootInfo.response'),
```

Comment:
`computed.readOnly` just prevents setting `response`, not properties on `response`.